templates/feature.html

```diff
@@ -25,7 +29,24 @@
       {% if feature.impl_status_chrome == "Removed" %} (removed){% endif %}
     </h2>
     <a href="/features#category: {{ feature.category }}" class="category">{{ feature.category }}</a>
-    {% if user.is_whitelisted %}<a href="/admin/features/edit/{{ feature.id }}" class="edit">&#160;edit</a>{% endif %}
+    {% if feature.meta.experimentalframework %}&nbsp;
```

Comment:
I've copied some of the div structure and CSS from the polymer element into here now, so the spacing is consistent. As you mentioned above, though, this should really be just *using* the `<chromedash-feature>` element. We should factor that out in the future.